handlers/geolocation/geolocation.go

```diff
@@ -36,8 +36,13 @@ func (c *GeolocationHandlersCollection) RedirectHandler() httprouter.Handle {
 		host := r.Host
 		pathType, prefix, playbackID, pathTmpl := parsePlaybackID(r.URL.Path)
 		redirectPrefixes := c.Config.RedirectPrefixes
-		lat := r.Header.Get("X-Latitude")
-		lon := r.Header.Get("X-Longitude")
+
+		query := r.URL.Query()
+		lat, lon := query.Get("lat"), query.Get("lon")
```

Comment:
nit: since these are query params, do we need some error checking to make sure they're in range? i.e. 
```
The numbers are in decimal degrees format and range from -90 to 90 for latitude and -180 to 180 for longitude
```

Reply:
Done! Since these are not critical inputs (same reason for not adding some security layer), I just ignore the values if they are invalid (incl from the headers). Updated PR